source/opt/private_to_local_pass.cpp

```diff
@@ -157,6 +157,12 @@ uint32_t PrivateToLocalPass::GetNewType(uint32_t old_type_id) {
 bool PrivateToLocalPass::IsValidUse(const Instruction* inst) const {
   // The cases in this switch have to match the cases in |UpdateUse|.
   // If we don't know how to update it, it is not valid.
+  if (inst->GetOpenCL100DebugOpcode() ==
+          OpenCLDebugInfo100DebugGlobalVariable ||
+      inst->GetOpenCL100DebugOpcode() == OpenCLDebugInfo100DebugDeclare ||
+      inst->GetOpenCL100DebugOpcode() == OpenCLDebugInfo100DebugValue) {
```

Comment:
Can a DebugDeclare and DebugValue reference a private varaible?  I thought these were only used for a local variable, and a private variable is a global variable.